controllers/ingress_controller_test.go

```diff
@@ -1,6 +1,18 @@
 package controllers
 
-import "testing"
+import (
+	"context"
+	"testing"
+
+	"github.com/TykTechnologies/tyk-operator/api/v1alpha1"
+	"github.com/TykTechnologies/tyk-operator/pkg/environmet"
+	"github.com/matryer/is"
+	v1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
 
 func TestShortHash(t *testing.T) {
 	if expect, got := "c09b5928e", shortHash("foo.com"+"/httpbin22222"); got != expect {
```

Comment:
I know that this test case does not belong to the PR, but we can add more "unhappy" cases here. For example, what happens if domain or host names are empty?

Reply:
Yes. We should do that.